### ISSUE TITLE
KafkaInputConfig with SASL_SCRAM

### ISSUE THREAD

**Issue by jrkinley** [bytewax/bytewax#107]
Allow `KafkaInputConfig` to authenticate over `SASL_SCRAM`.
```
Args
sasl-mechanism
    The authentication mechanism to use: SCRAM-SHA-256 or SCRAM-SHA-512
user
    SASL user to be used for authentication
password
    SASL password to be used for authentication
```

**Comment by awmatheson** [comment]
Thanks for adding this @jrkinley, these should be enabled with the changes made in #119 once it is merged and released.

**Comment by awmatheson** [comment]
Included in #119
